Initial commit;

610 opened the pull request:
Image Apache2.4 + PHP-FPM + MySQL client + compose + drush implemented

lmakarov commented:
Apache is constantly restarting. I'm seeing this in supervisord.log:

```
...
2015-03-05 02:58:50,696 INFO exited: apache2 (exit status 0; not expected)
2015-03-05 02:58:59,714 INFO spawned: 'apache2' with pid 260
2015-03-05 02:58:59,724 INFO exited: apache2 (exit status 0; not expected)
2015-03-05 02:59:09,750 INFO spawned: 'apache2' with pid 269
...
```

apache2.log:

```
... 
 * Starting web server apache2
 * 
 * Starting web server apache2
...
```

610 commented:
- Fixed constantly restarting apache, php-fpm
- Fixed both apache warning (ssl_module is already loaded, skipping and apache2: Could not reliably determine the server's fully qualified domain name)
- Removed startup.sh from supervisor

lmakarov commented:
Looks nice and clean now :)
Merging.
